Project-1/README.md

```diff
@@ -63,60 +68,228 @@ T	T	F	T	T
 
 4. Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   ```(a)  B that says that x is a brother of y
   (b)  A that says that x is an aunt of y
   (c)  C that says that x and y are cousins 
   (d)  O that says that x is an only child
   (e)  T that says that x has exactly two brothers 
-```
 
-5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+```(a) ΔI = x,y,z
+(a) parent_of = (z,x) (z,y)
+(a) Male = x
+(b) ΔI = w, x, y, z
+(b) praren_of = (w,x) (w,z) (z,y)
+(b) ¬Male = x
+(c) ΔI = v, w, x, y, z
+(c) parent_of = (v, w) (v,z) (w,x) (z,y)
+(d) Not possible in this language as saying x is an only child of y but would make everything else false.
+(e) Not possible in this laguage as you can say that x has two brothers but you can't say that x has exactly two brothers.  
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both.
+
+ TBox = {Brother ⊆ Male,		(TBox.1)
+	Aunt ⊆ ~Male}		(TBox.2)
+
+Donnie = {Frank: Brother ∩ Male		(Axiom.1)
+	Remi: Brother ∩ Male		(Axiom.2)
+	Mary: Aunt			(Axiom.3)
+	(Donnie, Frank): parent_of	(Axiom.4)
+	(Donnie, Remi): parent_of}		(Axiom.5)
+	
+ΔI = {w, x, y, z},
+William  = w
+Xaivar = x
+Eunice = y
+Zane = z
+Male = {w, x}
+Brother = {w, x}
+Aunt = {y}
+Parent_of = {(z,w), (z,x)}
 
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```(a) ∃x∀y and ∀x∃y
+ ```(a) ∃x∀y and ∀x∃y
   (b) ∃x∀y∃z and ∀x∃y∀z 
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
-```
-	
+
+```(a) ∃x∀y says something like there is some x such that for all y
+(a) ∀x∃y says for all x there is some y 
+(b) ∃x∀y∃z says there is some x such that for all y there is some z such that 
+(b) ∀x∃y∀z says for all x there is come y such that for all x
+(c) ∀x∃y∀z∃w says for all x there is some y such that for all z there is some w
+(c) ∃x∀y∃z∀w says there is some x such that for all y there exists some z such that for all w
+
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
-```
+
+``` Here is my diagram of the universe: [label](file:///Universe.pub)
+U: {0,1}
+R: {(0,0), (0,1), (1,1)} 
```

Comment:
It seems that your graph satisfy two formulas at the same time. In order to differ between them, maybe you need a graph including 3 distinct points.